schema/samcli.json

```diff
@@ -1206,9 +1247,60 @@
           "properties": {
             "parameters": {
               "title": "Parameters for the traces command",
-              "description": "Available parameters for the traces command:\n",
+              "description": "Available parameters for the traces command:\n* trace_id:\nFetch specific trace by providing its id\n* output:\nThe formatting style of the command output. Following options are available:\n\nTEXT: Prints information as regular text with some formatting (default option)\n\nJSON: Prints each line as JSON without formatting\n* tail:\nTail events. This will ignore the end time argument and continue to fetch events as they become available. If option --tail without a --name will pull from all possible resources\n* end_time:\nFetch events up to this time. Time can be relative values like '5mins ago', 'tomorrow' or formatted timestamp like '2018-01-01 10:10:10'\n* start_time:\nFetch events starting at this time. Time can be relative values like '5mins ago', 'yesterday' or formatted timestamp like '2018-01-01 10:10:10'. Defaults to '10mins ago'.\n* beta_features:\nEnable/Disable beta features.\n* debug:\nTurn on debug logging to print debug message generated by AWS SAM CLI and display timestamps.\n* profile:\nSelect a specific profile from your credential file to get AWS credentials.\n* region:\nSet the AWS Region of the service. (e.g. us-east-1)",
               "type": "object",
-              "properties": {}
+              "properties": {
+                "trace_id": {
+                  "title": "trace_id",
+                  "type": "string",
+                  "description": "Fetch specific trace by providing its id"
+                },
+                "output": {
+                  "title": "output",
+                  "type": "string",
+                  "description": "The formatting style of the command output. Following options are available:\n\nTEXT: Prints information as regular text with some formatting (default option)\n\nJSON: Prints each line as JSON without formatting",
+                  "enum": [
+                    "json",
+                    "text"
```

Comment:
question: Is this case sensitive? The description is upper case, but then the enum is lower case. If it doesn't matter, then I think that this is fine as is.

Reply:
Testing it quickly, it seems like case sensitivity doesn't affect it; it works with all of `["json", "JSON", "jSoN"]` when I tested.